src/TTextEdit.cpp

```diff
@@ -1160,28 +1162,43 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
 
 
         QAction* action = new QAction("copy", this);
-        action->setStatusTip(tr("copy selected text to clipboard"));
+        action->setToolTip(tr("copy selected text to clipboard"));
         connect(action, SIGNAL(triggered()), this, SLOT(slot_copySelectionToClipboard()));
         QAction* action2 = new QAction("copy HTML", this);
-        action2->setStatusTip(tr("copy selected text with colors as HTML (for web browsers)"));
+        action2->setToolTip(tr("copy selected text with colors as HTML (for web browsers)"));
         connect(action2, SIGNAL(triggered()), this, SLOT(slot_copySelectionToClipboardHTML()));
         QAction* action3 = new QAction("select all", this);
-        action3->setStatusTip(tr("select all text in the buffer"));
+        action3->setToolTip(tr("select all text in the buffer"));
         connect(action3, SIGNAL(triggered()), this, SLOT(slot_selectAll()));
 
         QString selectedEngine = mpHost->getSearchEngine().first;
         QAction* action4 = new QAction(("search on " + selectedEngine), this);
         connect(action4, SIGNAL(triggered()), this, SLOT(slot_searchSelectionOnline()));
-        action4->setStatusTip("search selected text on " + selectedEngine);
+        action4->setToolTip(tr("search selected text on %1").arg(selectedEngine));
 
         auto popup = new QMenu(this);
+        popup->setToolTipsVisible(true); // Not the default...
```

Comment:
Unfortunately with this option set then the default tooltip for any QAction on a QMenu is _to repeat the menu text_ - so either we have short tool tips on ALL items or we have no tooltips - although let me just check to see what happens if I explicitly set a tool-tip to an empty string rather then a null one...